src/test/resources/org/gradle/profiler/report/example.json

```diff
@@ -1,15 +1,45 @@
 {
   "environment": {
-    "profilerVersion": "0.15.0-alpha2"
+    "profilerVersion": "0.15.0-alpha2",
+    "operatingSystem": "mac os x x86_64 10.15.6",
+    "date": "2020-09-13T12:26:40Z"
   },
   "scenarios": [
     {
       "definition": {
         "name": "no-optimizations",
         "title": "Baseline",
         "displayName": "Baseline using Gradle 6.7-20200903220111+0000",
-        "buildTool": "6.7-20200903220111+0000",
+        "buildTool": "Gradle 6.7-20200903220111+0000",
         "tasks": ":santa-tracker:assembleDebug",
+        "version": "6.7-20200903220111+0000",
+        "gradleHome": "/home/lptr/Workspace/santa-tracker-performance/gradle-user-home/wrapper/dists/gradle-6.7-20200903220111+0000-bin/6csfyoy16renvfjrtt5md548g/gradle-6.7-20200903220111+0000",
+        "javaHome": "/opt/files/jdk-linux/OpenJDK8U-jdk_x64_linux_hotspot_8u242b08.tar.gz",
+        "usesScanPlugin": false,
+        "action": "run tasks :santa-tracker:assembleDebug",
+        "cleanup": "do nothing",
+        "invoker": "Tooling API",
+        "mutator": "ApplyNonAbiChangeToSourceFileMutator(santa-tracker-java/village/src/main/java/com/google/android/apps/santatracker/village/SnowFlake.java)",
+        "args": [
+          "-Dorg.gradle.workers.max\u003d8",
```

Comment:
`0x3D` is the ASCII/UTF-8 code for `=`, Gson escapes it because it's an HTML entity:

> By default, Gson escapes HTML characters such as `<` `>` etc.

https://www.javadoc.io/doc/com.google.code.gson/gson/2.7/com/google/gson/GsonBuilder.html#disableHtmlEscaping--

I think it's fine to leave it like this, especially that we'll be embedding the output in HTML.

Reply:
👍